ReactTask/src/components/CardPatientProfileDoctor/CardPatientProfileDoctor.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  StylSubTitleCard,
+  StylTitleCard,
+  StylBoxImgPatient,
+  StylBoxColumnInfo,
+  StylBoxRowLine,
+  StylBoxCardPatient,
+  StylBoxDoctor,
+  BoxDataPatient,
+  BoxInfoPatient,
+  BtnMore,
+} from './../index';
+
+import scheduleImg from './../../img/icons/schedule.png';
+import BoxPatientsList from '../common/Boxes/StylBoxPatientsList';
+
+const CardPatientProfileDoctor = ({ dataPatients, imgPatientsArr }) => {
+  return (
+    <BoxPatientsList>
+      {dataPatients.map((item) => (
+        <StylBoxCardPatient key={item.id}>
+          <StylBoxRowLine>
+            <StylBoxDoctor>
+              <StylBoxImgPatient
+                src={imgPatientsArr[item.id - 1]}
+                alt={item.alt}
+              />
+              <StylBoxColumnInfo>
+                <StylTitleCard>{item.name}</StylTitleCard>
+                <StylSubTitleCard status={item.status}>
+                  {item.statusText}
+                </StylSubTitleCard>
+              </StylBoxColumnInfo>
+            </StylBoxDoctor>
+            <BtnMore />
+          </StylBoxRowLine>
+          <BoxDataPatient data={item.data} />
+          <BoxInfoPatient info={item.info} imgWay={scheduleImg} />
+        </StylBoxCardPatient>
+      ))}
+    </BoxPatientsList>
```

Comment:
What is the difference between patient and doctor? Can you merge functional into one compoent. That will accept patients or doctords.

Reply:
The two components have a difference in status (confirmed, canceled, pending) and an icon (heart, list)

Reply:
I was talking about code duplicates in both components, you have almost the same Header across your app. Avoid code duplicates, for this cases we have components to reuse.

Reply:
Created 1 component from 2 card `CardPatientProfile`